docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,466 @@
+---
+title: Getting started with KLT
+description: Get started with the Keptn Lifecycle Toolkit
+weight: 10
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+Keptn Lifecycle Toolkit works whether or not you use a GitOps strategy.
+The following is an imperative walkthrough.
+
+## Prerequisites
+
+- A Kubernetes cluster >= 1.24 (we recommend [Kubernetes kind](https://kind.sigs.k8s.io/docs/user/quick-start/))
+  (`kind create cluster`)
+- [Helm](https://helm.io) CLI available
+
+## Objectives
+
+- Install Keptn Lifecycle Toolkit on your cluster
+- Annotate a namespace and deployment to enable Keptn Lifecycle Toolkit
+- View DORA Metrics
+- Install Grafana and Observability tooling to view DORA metrics
+
+## System Overview
+
+By the end of this page, here is what will be built.
+This system will be built in stages.
+
+![system overview](/docs/install/assets/install01.png)
+
+## The Basics: A Deployment, Keptn and DORA Metrics
+
+Let's start with the basics.
+Here is what we will now build.
+
+A deployment will occur.
+Keptn will monitor the deployment and generate:
+
+- An OpenTelemetry trace per deployment
+- DORA metrics
+
+![the basics](/docs/install/assets/install02.png)
+
+Notice though that the metrics and traces have nowhere to go.
+That will be fixed in a subsequent step.
+
+## Step 1: Install Keptn Lifecycle Toolkit
+
+Install Keptn Lifecycle Toolkit using Helm:
+
+```shell
+helm repo add klt https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn klt/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+Keptn will need to know where to send OpenTelemetry traces.
+Of course, Jaeger is not yet installed so traces have nowhere to go (yet),
+but creating this configuration now means the system is preconfigured.
+
+Save this file as `collectorconfig.yaml`:
```

Comment:
Should we advise them where to put this?  Perhaps a suggestion that they create a directory on their workstation and put all these files there?  It seems like, in real life, they would subdivide that directory but maybe it's not necessary to introduce that complexity here.  And yes, they should know better than to just start creating these files in $HOME but...  And, for someone who has never explicitly worked with k8s before, it might not be obvious that you create all these files on your workstation.  Just saying -- a simple suggestion might be useful.

Reply:
I've added words to this effect.